src/TextShow/Data/Aeson.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# OPTIONS -fno-warn-orphans #-}
+import Data.Aeson
```

Comment:
Don't forget the `module TextShow.Data.Aeson` part—its absence is causing the most recent CI failures.

Reply:
no sure what i was doing...